sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -937,6 +937,15 @@ object SQLConf {
       .timeConf(TimeUnit.SECONDS)
       .createWithDefault(0L)
 
+  val THRIFTSERVER_FORCE_CANCEL =
+    buildConf("spark.sql.thriftServer.forceCancel")
+      .doc("When true, all the job of query will be cancelled and running tasks will be" +
+        "interrupted. When false, all the job of query will be cancelled but running task" +
+        "will be remained until finished.")
```

Comment:
Hmm, is this doc correct? What it means all the job of query will be cancelled when this is true?

Reply:
Updated.